packages/fxa-auth-client/browser.ts

```diff
@@ -1,4 +1,5 @@
 import AuthClient from './lib/client';
+export * from './lib/crypto';
```

Comment:
In `pages/Signup/container.tsx`, we're importing this directly from `fxa-auth-client/lib/crypto` instead of `fxa-auth-client/browser`. I think your way is probably better for now but (not in this PR) we should update that over on Signup for consistency. I wonder also if we should move this somewhere in `fxa-settings` itself.